src/internal/m365/service/groups/restore.go

```diff
@@ -39,14 +40,9 @@ func ConsumeRestoreCollections(
 		caches         = drive.NewRestoreCaches(backupDriveIDNames)
 		lrh            = drive.NewLibraryRestoreHandler(ac, rcc.Selector.PathService())
 		el             = errs.Local()
+		siteNames      = map[string]string{}
```

Comment:
Nit: trying for a new standard here.  Maps should be named either for their k:v pairs for maximum clarity.
```suggestion
		webURLToSiteNames      = map[string]string{}
```